modules/create-policy-from-system-policies-view.adoc

```diff
@@ -43,12 +43,21 @@ However, you cannot use regular expressions for selecting deployments.
 The *Excluded Images* setting only applies when you check images in a continuous integration system with the *Build* lifecycle stage.
 It will not have any effect if you use this policy to check running deployments in the *Deploy* lifecycle stage or runtime activities in the *Runtime* lifecycle stage.
 ====
+. Optional: Under the *MITRE ATT&CK* section, select the *Tactics* and the *Techniques* you want to specify for the policy.
+.. Click *Add tactics* and then select a tactic from the dropdown list.
```

Comment:
```suggestion
.. Click *Add tactics*, and then select a tactic from the dropdown list.
```

ISG requires a comma between independent clauses. See pp. 85-86.

Reply:
Also, from the screenshots this appears to be **Add tactic** rather than **Add tactics**.